core/commands/object/object.go

```diff
@@ -342,6 +344,7 @@ And then run:
 	},
 	Options: []cmds.Option{
 		cmds.StringOption("inputenc", "Encoding type of input data, either \"protobuf\" or \"json\"."),
+		cmds.StringOption("datafieldenc", "Encoding type of the data field, either \"text\" or \"base64\"."),
```

Comment:
add `.Default("text")` to the end of this option, so you don't have to do the `!found` check later